File: countingSort/countingSort.js
const arr = [0, 5, 7, 20, 99, 99, 99, 0, 0]


function countingSort(arr){
    let bank = []
    
    for (i=0;i<100;i++){
        bank.push(0)
    }
    for (i=0;i<arr.length;i++){
        let change = arr[i]
        bank[change] = bank[change] + 1
    }
    console.log(bank);
    return bank
}

countingSort(arr)

File: countingSort/counting_sort.py
arr = [0, 99, 99, 99, 99]

def counting_sort(arr):
    bank = []

    for i in range(100):
        bank.append(0)

    for i in arr:
        bank[i] = bank[i] + 1
    
    print(bank)
    return bank



counting_sort(arr)
